### PR TITLE
Update osd-network-verifier to Go 1.19

### PR DESCRIPTION
**Pull request by mjlshen**
The culmination of efforts starting with #166 - no other dependency updates

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
 	github.com/aws/smithy-go v1.9.0
-	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/mock v1.6.0
 	github.com/openshift-online/ocm-sdk-go v0.1.224
 	github.com/spf13/cobra v1.4.0
 	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
 	google.golang.org/api v0.84.0
 )
+
+require (
```
Comment (boranx) [inline go.mod:18]: hmm, any idea why this is duplicating `require`? It seems like the ones come with the update is not being used by the verifier, such as `grpc ` or `protobuf `

Comment (mjlshen) [reply go.mod:18]: Ah it separates out direct/indirect dependencies by default now, I want to say starting Go 1.17? But e.g. https://github.com/openshift/osdctl/blob/master/go.mod

Comment (mjlshen) [reply go.mod:18]: found it! It is Go 1.17 https://github.com/golang/go/issues/47733

**Review by abyrne55 — APPROVED** [review]
/lgtm
/approve

Tested locally by building and running standalone CLI tool's egress test. Thanks, @mjlshen!

**Review by boranx — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**